projects/storefrontlib/src/cms-components/organization/index.ts

```diff
@@ -1,2 +1 @@
-export * from './abstract-component/index';
```

Comment:
Can we drop `cms-components/organization` folder in storefrontlib together with the unused fake-tabs component?